Automattic/jetpack
# WAF: Add unit tests to confirm removal on Atomic

See https://github.com/Automattic/jetpack/pull/26401#issuecomment-1257118898

Moves the check to the 3rd-party file and adds unit tests to prevent future regressions.

#### Changes proposed in this Pull Request:

* Moves the WAF-Atomic check to the 3rd-party file with other Atomic checks.
* Adds unit testing.

#### Other information:


#### Jetpack product discussion


https://github.com/Automattic/jetpack/pull/26401#issuecomment-1257118898

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* On an Atomic dev site, load this branch and confirm the WAF is not available.
* Passing unit tests.
